lib/simple_sqs/worker.rb

```diff
@@ -20,6 +20,11 @@ def initialize queue_url:
     )
     @processor = SimpleSqs::Processor.new
     @poller = Aws::SQS::QueuePoller.new(@queue_url, {client: @client})
+    @transaction = (ENV.fetch('SIMPLE_SQS_NO_AR_TRANSACTION', true) == true)
```

Comment:
Figuring out that this will never be `true` took me a while. How about:
```suggestion
    @transaction = !ENV.has_key?('SIMPLE_SQS_NO_AR_TRANSACTION')
```
?

Reply:
Yep, looks much better!